validate/error.go

```diff
@@ -0,0 +1,69 @@
+package validate
```

Comment:
> The runtime-tool and image-tool are separated, where should we have such a 'stand-alone module', ...

I was thinking a separate sub-package in this repo.  Or maybe in runtime-spec, since both image repos currently have some runtime-spec connections.  It seems too tiny for its own repo, and opencontainers/certification seems more focused on policy than implementation.